.circleci/config.yml

```diff
@@ -324,6 +324,13 @@ jobs:
       - checkout
       - setup_remote_docker:
           docker_layer_caching: true
+      - attach_workspace:
+          at: /tmp/generated-sql
+      - run:
+          name: Move generated-sql into place
+          command:
+            rm -rf sql/
+            cp -r /tmp/generated-sql/sql sql
```

Comment:
This `deploy` step may turn out to be a good integration point for private-bigquery-etl code as well. The [proposal for private ETL code](https://docs.google.com/document/d/1GuuxseYBbUYiq-k4ut8LOEeVBtwwDB3cEBf35NmbTFE/edit#) currently discussed building a separated docker container, but @whd pointed out that consolidating artifacts is preferable from an ops management perspective.

Reply:
I'm generally supportive of considerable simplification.